tests/h/accounts/schemas_test.py

```diff
@@ -665,6 +665,59 @@ def schema(self):
         return schemas.CreateUserAPISchema()
 
 
+class TestUpdateUserAPISchema(object):
+    def test_it_raises_when_email_empty(self, schema, payload):
```

Comment:
Hmm ... these tests duplicate quite a lot of the code for the above tests for creating users. It would be nice to avoid that.

Reply:
IMO it doesn't make much sense to de-duplicate the tests if the schema implementation is duplicated as well. But there's probably something to look into in the future to de-duplicate the schemas and also linking them from the API docs, so we don't have to remember to update them every time.